ODIN_II/SRC/output_blif.cpp

```diff
@@ -211,6 +211,7 @@ void output_blif(const char *file_name, netlist_t *netlist)
 			}
 
 		}
+		free_nnode(node);
```

Comment:
Freeing the node here will cause the output node to be freed, I would caution you to either free the whole netlist (nodes, net and pins) or none as we are currently doing it.

This one should be a separate PR. @ademmings

Reply:
Idk if worrying about the whole netlist is all that relevant - the pointer to this node is no longer pointing to it (is it getting replaced somewhere?) and that's the root problem. I agree with the separate PR, but I would take a different approach and try to track down where the node is actually getting lost/replaced (I don't immediately see it in this file, might have to do some digging in other files)

Reply:
I removed this change from the PR. I'll try to track down where the pointer is being replaced and try to free it there